### ISSUE TITLE
[BUG] Metadata sheet ISA terms are sometimes nested, sometimes not

### ISSUE THREAD

**Issue by omaus** [nfdi4plants/ISADotNet#138]
**Describe the bug**
In XLSX metadata sheets, terms are sometimes nested and sometimes not:

Nested:
![image](https://github.com/nfdi4plants/ISADotNet/assets/47781170/ee53e001-83de-4ab3-b5b7-c0e9c7968370)
Here: Investigation file: e.g. `INVESTIGATION` -> `INVESTIGATION PUBLICATIONS` -> `Investigation Publication Title`

Not nested:
![image](https://github.com/nfdi4plants/ISADotNet/assets/47781170/f81bfe92-048f-4cf5-922a-baf6011ce3e2)
Here: Assay file: e.g. `ASSAY PERFORMERS` -> `First Name`

**Expected behavior**
Shouldn't this be nested as well in Assay file? Or not nested in Investigation file and Study file?

CC: @kMutagene

**Comment by kMutagene** [comment]
> Shouldn't this be nested as well in Assay file? Or not nested in Investigation file and Study file?

IIRC the investigation file looks exactly the same way that files created by original isa tooling look like. For the study metadata section (which is new in the arc, not included in original isa) we follow this scheme, but break it in the assay file.
 
So i think the assay metadata section should look like this:

```
ASSAY METADATA
Asssay Measurement Type
Asssay Measurement Type Term Accession Number
Asssay Measurement Type Term Source REF
Asssay Technology Type
Asssay Technology Type Term Accession Number
Asssay Technology Type Term Source REF
Asssay Technology Platform
Asssay File Name
ASSAY PERFORMERS
Asssay  Performer Last Name
Asssay  Performer First Name
Asssay  Performer Mid Initials
Asssay  Performer Email
Asssay  Performer Phone
Asssay  Performer Fax
Asssay  Performer Address
Asssay  Performer Affiliation
Asssay  Performer Roles
Asssay  Performer Roles Term Accession Number
Asssay  Performer Roles Term Source REF
```

**Comment by kMutagene** [comment]
For future reference, I found more inconsistencies:

- the metadata sheet names are different: for Investigation, it is `isa_investigation`, while for Study/Assay it is `Study`/`Assay` 
  --> i propose `isa_assay`/`isa_study` for Study/Assay as names for metadata sheets

- while the investigation metadata section just has the `INVESTIGATION` header (followed by identifier etc.), it is again different for Study/Assay metadata sheets, where it is `STUDY METADATA`/`ASSAY METADATA`.
  --> I propose   `STUDY`/`ASSAY` for Study/Assay as headers for Study/Assay metadata sheets.

Basically, all suggestions boil down to adhereing to the scheme of the original investigation metadata sheet, since the study/assay metadata sheets are our own additions to the standard.

**Comment by Freymaurer** [comment]
The logic for this can be found in ISA.Spreadsheet 👍

**Comment by kMutagene** [comment]
Seems like some of the issues are already fixed, i just added metadata sheet names in #140 .

**Comment by kMutagene** [comment]
JFC, seems like this is the current state when my PR is merged:

### Study
- [x] correct sheet name (isa_study)
- [x] correct header (STUDY) 

### Assay
- [x] correct sheet name (isa_assay)
- [x] correct header (ASSAY) 
- [ ] correct nesting headers (e.g. Measurement type -> Assay Measurement Type)

**Comment by HLWeil** [comment]
@kMutagene @omaus @Freymaurer 
I fixed this in #142 

Tables now follow standard isa:
```
ASSAY
Assay Measurement Type
Assay Measurement Type Term Accession Number
Assay Measurement Type Term Source REF
Assay Technology Type
Assay Technology Type Term Accession Number
Assay Technology Type Term Source REF
Assay Technology Platform
Assay File Name
ASSAY PERFORMERS
Assay Person Last Name
Assay Person First Name
Assay Person Mid Initials
Assay Person Email
Assay Person Phone
Assay Person Fax
Assay Person Address
Assay Person Affiliation
Assay Person Roles
Assay Person Roles Term Accession Number
Assay Person Roles Term Source REF
```

Tested for reading fixed and deprecated, and writing fixed.
